### DISCUSSION TITLE
New meeting type: Lecture

### DISCUSSION THREAD

**Discussion by MikeTheFid** [code4recovery/12-step-meeting-list#899] [Q&A] [answered]
I was asked to add a new meeting recently, and the organizer said it was an "Open Lecture" meeting. Not sure what that is or means, but perhaps you could add **Lecture** to the types list.

Cheers,
Mike

**Comment by joshreisner** [comment, marked as answer]
Hi Mike, we follow GSO's lead now with regard to AA meeting types. If you want you could reach out to app support (appsupport@aa.org), but in my experience they have not added any new types.

If you want you can check out the first item in our FAQ for [instructions on how to add a custom type to your website](https://wordpress.org/plugins/12-step-meeting-list/#my%20meeting%20type%20isn%27t%20listed%21).

When picking a code, please be sure to pick one that's not already on [this list of types currently in use](https://github.com/code4recovery/spec#meeting-types). Doing so would cause that type to show up in the app. That's currently happening in some places 😢 .

**Reply by joshreisner** [reply]
honestly my recommendation though would be to indicate that in the meeting notes. types are really supposed to just be an aid to someone looking for a particular type of meeting, eg Women, and I'm struggling to imagine someone looking for a "Lecture" meeting.

the "lecture" idea sounds like "Speaker" to me, so perhaps that's an option here?

or if the information is for newcomers, perhaps "Newcomer"?

**Reply by MikeTheFid** [reply]
Hi Josh, I had added "Lecture" to the meeting notes. _Thank you for your help!_
